### PR TITLE
Add Currencies and Exponents to SDK

### PR DESCRIPTION
**Pull request by mike-brookes**
Addition of a new Currencies class with available Currency Codes and their Exponents to assist users of the SDK in formatting currencies correctly for use with the API

### CODE REVIEW

File: src/API/Util/Currencies.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace OnPay\API\Util;
+
+use OnPay\API\Exception\ApiException;
+
+class Currencies {
+    /**
```
Comment (ebbesmoeller) [inline src/API/Util/Currencies.php:8]: Indication of what ISO standard these currency codes are from, would be appreciated.

File: src/API/Util/Currencies.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+     * @throws ApiException
+     */
+    public function getCurrencyExponent($currencyCode) {
+        $currencyArray = self::ACCEPTED_CURRENCIES;
+        if (!isset($currencyArray[$currencyCode])) {
+            throw new ApiException("Invalid Currency Code: $currencyCode - No exponent available.");
+        }
+        return $currencyArray[$currencyCode];
+    }
+
```
Comment (ebbesmoeller) [inline src/API/Util/Currencies.php:49]: I think an additional function to validate if currency is accepted would be useful here.
taking the currency code as a parameter according to ISO4217. Perhaps for both numeric and alpha3 codes?

File: src/API/Util/Currency.php [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    private function assignValuesFromAlpha3() {
+        $this->iso4217 = self::ALPHA3_CURRENCIES[$this->alpha3]['iso4217'];
+        $this->exponent = self::ALPHA3_CURRENCIES[$this->alpha3]['exponent'];
+    }
+
+    private function assignValuesFromIso4217() {
+        $this->alpha3 = self::ISO4217_CURRENCIES[$this->iso4217]['alpha3'];
+        $this->exponent = self::ISO4217_CURRENCIES[$this->iso4217]['exponent'];
+    }
+}
```
Comment (dvaeversted) [inline src/API/Util/Currency.php:71]: Aren't we missing a new line here ?

File: src/API/Util/Currencies.php [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+        'ECP' => array('exponent'=>2, 'iso4217' => 818),
+        'GBP' => array('exponent'=>2, 'iso4217' => 826),
+        'USD' => array('exponent'=>2, 'iso4217' => 840),
+        'EUR' => array('exponent'=>2, 'iso4217' => 978),
+        'UAH' => array('exponent'=>2, 'iso4217' => 980),
+        'PLN' => array('exponent'=>2, 'iso4217' => 985),
+        'BRL' => array('exponent'=>2, 'iso4217' => 986),
+    );
+
+    const ISO4217_CURRENCIES = array(
```
Comment (dvaeversted) [inline src/API/Util/Currencies.php:33]: I am not a big fan of having the same static data 2 times, is it possible to solve this in a different way to ensure an update is never "halfly" done ?

**Review by ebbesmoeller — APPROVED** [review]

**Pull request merged**